utils/protobuf-schema-utilities/src/main/proto/decimal/decimal.proto

```diff
@@ -0,0 +1,39 @@
+syntax = "proto3";
+
+package decimal;
+
+option java_package = "decimal";
+option java_outer_classname = "ProtobufSchemaDecimal";
+
+// Protobuf standard library / WellKnownTypes don't provide a Decimal type.
+// This posses a challenge to represent Decimals that cannot be accurately represented
+// using float or double types. This Proto provides a way to represent Decimals for
+// customers using Glue Schema Registry.
```

Comment:
Glue Schema Registry is not relevant here as this is general purpose Apicurio library. Please remove all other references.